resolvers/relay.ts

```diff
@@ -0,0 +1,66 @@
+export type RelayInformation = {
```

Comment:
rename the name of the file to either `nip11.ts` or `relay-information.ts`

Reply:
Renamed to `nip11.ts`.